bignum.c

```diff
@@ -204,6 +204,20 @@ void bn_zero(bignum256 *a)
 	}
 }
 
+// sets a bignum to one.
+void bn_one(bignum256 *a)
```

Comment:
I haven't looked through source code fully yet, so I didn't know if `inline` functions were in use already. The point I was making is that you need either GNU89 or C99 for `inline` (and we have both :smile:)